vtex-sites/gatsby.store
# Feat: Uses `ProductCard` component from FSUI

## What's the purpose of this pull request?

This PR is part of the Component Placement project. Replaces `Card` for `ProductCard`

## How does it work?

- Uses `ProductCard` component from FSUI instead of `Card`
- Renames  `ProductCard` prop `actionabled` to `actionable`

## How to test it?

- Nothing should change visually on the component.
- Check on the [preview](https://preview-34--gatsby.preview.vtex.app) if the `ProductCard` is working fine as [before](https://gatsby.vtex.app/). 

- The `ProductCard` component appear on the **[Homepage](https://preview-34--gatsby.preview.vtex.app)** and **[PLP](https://preview-34--gatsby.preview.vtex.app/office)**:

|Homepage - ProductShelf|
|-|
![image](https://user-images.githubusercontent.com/3356699/168401467-8a63c660-c05a-4996-ad6c-376faa955f47.png)

|Homepage|
|-|
![image](https://user-images.githubusercontent.com/3356699/168401491-df6e981c-bee4-4c6d-a177-ea325f208fbe.png)

|PLP - Product Listing Page|
|-|
![plp](https://user-images.githubusercontent.com/3356699/168401530-6490465b-eee3-4c37-96ce-deea2e712aa9.png)


## Checklist

<em>You may erase this after checking them all ;)</em>
- [x] After https://github.com/vtex/faststore/pull/1272 is merged, update faststore/ui version on stores and merge those PRs as well
